libr/bin/format/mach0/mach0.c

```diff
@@ -3159,16 +3158,10 @@ static void parse_relocation_info(struct MACH0_(obj_t) *bin, RSkipList * relocs,
 RSkipList *MACH0_(get_relocs)(struct MACH0_(obj_t) *bin) {
 	RSkipList *relocs = NULL;
 	RPVector *threaded_binds = NULL;
-	ulebr ur = {NULL};
 	size_t wordsize = get_word_size (bin);
 	if (bin->dyld_info) {
-		ut8 *opcodes,*end, type = 0, rel_type = 0;
-		int lib_ord, seg_idx = -1, sym_ord = -1;
-		char *sym_name = NULL;
-		size_t j, count, skip, bind_size, lazy_size, weak_size;
-		st64 addend = 0;
-		ut64 segmentAddress = 0LL;
-		ut64 addr = 0LL;
+		ut8 *opcodes, rel_type = 0;
```

Comment:
it's a pointer, so it must use NULL, not 0

Reply:
Only `opcodes` is, `rel_type` is a `ut8`.